src/sdk-reference/cpp/1/websocket/remove-all-listeners/index.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: sdk.html.hbs
```

Comment:
Please apply to this page all the suggestions I made for the same method documentation in the Protocol class